Strategies/IITB_KPIs.md

```diff
@@ -0,0 +1,74 @@
+
+`ratio = service requests / service personnel`
+
+## 4. **Percentage of IITB services made available as SaaS**
+
+IITB services currently offered, as catalogued in the ITSM Repository, will be offered as SaaS for IITB consumers. This transformation to SaaS provisioning and maintenance will be measured by the rate at which those services are changed to self-served ones. The measurement will look at [IITB's Service Catalogue](http://iservice.prv/eng/service_catalogues/imit_services.shtml), which is being modernized for 2020, as reference point and monitor their transformation to SaaS.
+
+`percentage = SaaS / total ITSM services`
+
+## 5. **Availability uptime of ESDC core systems**
+
+Moving IITB into a service-oriented organization will require IITB to manage the complexity and growth of its services pro-actively. The measurement of availability is driven by time loss whereas the measurement of reliability is driven by the frequency and impact of failures. This indicator focuses on availability as it's what clients of the service are concerned with. The ***service's*** maximum allowable downtime, as agreed by the BCP Business Impact Analysis exercise, will be used to assess IITB's service availability. The Core systems will be the ones flagged as *high business value* within [ESDC's Application Portfolio Management (APM) program](http://dialogue/grp/PR6303013/APM_Program/default.aspx).
+
+`service availability = (total elapsed time - sum of downtime) / total elapsed time`
+
+*Note:* Services are never expected to be up 100% of the time. [See here](http://www.gcpedia.gc.ca/wiki/OCIO_Application_Portfolio_Management/Application_Portfolio_Management_User_Guide#Application_Mission_Criticality_and_Critical_Services) for more details.  
+
+## 6. **Increase rate of project intake by IITB**
+
+IITB's project intake capacity will be measured by how much it improves, fostering a continuous improvement measurement over a hard target. The measurement will be on the rate at which intake capacity increases. The PMO's [Active Project List](http://dialogue/grp/ippst/Approved%20Projects%20Lists%20%28APL%29/Forms/AllItems.aspx?RootFolder=%2Fgrp%2Fippst%2FApproved%20Projects%20Lists%20%28APL%29%2FFY%202019%2D2020&FolderCTID=0x012000D8E2E348AB0A034886495AE4BED12E07&View=%7b2C779E42-EB28-444B-AACD-39CC61DDBCE2%7d), for Major/Minor/Small ESDC projects, as well as the PMO's [IITB Lite projects](http://dialogue/grp/ITPM-GPTI/Lists/IITBPMP%20Small%20Projects/AllItems.aspx) will be used as reference points to monitor intake increase.
+
+`rate of intake = (Projects Year_n - Projects Year_n-1) / Projects Year_n-1`
+
+## 7. **Reduction in Cyber-incidents**
+
+This indicator will monitor the number of cyber-incidents across ESDC and measure the rate at which they decrease.
+
+`rate = (Incidents Year_n - Incidents Year_n-1) / Incidents Year_n-1`
+
+## 8. **Client Satisfaction of IITB Services**
```

Comment:
Well, that one is subjective but if we can move the needle to the right, that will be good!